src/main/resources/messages_vi_VN.properties

```diff
@@ -0,0 +1,1052 @@
+pageLayout.tags=ghép,tổng hợp,xem đơn,sắp xếp
+
+home.scalePages.title=Điều chỉnh kích thước/tỷ lệ trang
+home.scalePages.desc=Thay đổi kích thước/tỷ lệ của trang và/hoặc nội dung của nó.
+scalePages.tags=điều chỉnh kích thước,sửa đổi,kích thước,điều chỉnh
+
+home.pipeline.title=Pipeline (Nâng cao)
+home.pipeline.desc=Chạy nhiều thao tác trên PDF bằng cách định nghĩa các tập lệnh pipeline
+pipeline.tags=tự động hóa,chuỗi,kịch bản,xử lý hàng loạt
+
+home.add-page-numbers.title=Thêm số trang
+home.add-page-numbers.desc=Thêm số trang xuyên suốt tài liệu ở vị trí cố định
+add-page-numbers.tags=đánh số trang,gắn nhãn,sắp xếp,chỉ mục
+
+home.auto-rename.title=Tự động đổi tên tệp PDF
+home.auto-rename.desc=Tự động đổi tên tệp PDF dựa trên tiêu đề được phát hiện
+auto-rename.tags=tự động phát hiện,dựa trên tiêu đề,sắp xếp,đổi nhãn
+
+home.adjust-contrast.title=Điều chỉnh màu sắc/tương phản
+home.adjust-contrast.desc=Điều chỉnh độ tương phản, độ bão hòa và độ sáng của PDF
+adjust-contrast.tags=hiệu chỉnh màu sắc,điều chỉnh,sửa đổi,nâng cao
+
+home.crop.title=Cắt PDF
+home.crop.desc=Cắt PDF để giảm kích thước (giữ nguyên văn bản!)
+crop.tags=cắt tỉa,thu nhỏ,chỉnh sửa,định hình
+
+home.autoSplitPDF.title=Tự động tách trang
+home.autoSplitPDF.desc=Tự động tách PDF đã quét với mã QR tách trang quét vật lý
+autoSplitPDF.tags=Dựa trên QR,tách,phân đoạn quét,sắp xếp
+
+home.sanitizePdf.title=Làm sạch
+home.sanitizePdf.desc=Xóa các tập lệnh và phần tử khác khỏi các tệp PDF
+sanitizePdf.tags=làm sạch,bảo mật,an toàn,loại bỏ mối đe dọa
+
+home.URLToPDF.title=URL/Trang web sang PDF
+home.URLToPDF.desc=Chuyển đổi bất kỳ URL http(s) nào thành PDF
+URLToPDF.tags=chụp web,lưu trang,web sang tài liệu,lưu trữ
+
+home.HTMLToPDF.title=HTML sang PDF
+home.HTMLToPDF.desc=Chuyển đổi bất kỳ tệp HTML hoặc zip nào thành PDF
+HTMLToPDF.tags=đánh dấu,nội dung web,chuyển đổi,chuyển
+
+home.MarkdownToPDF.title=Markdown sang PDF
+home.MarkdownToPDF.desc=Chuyển đổi bất kỳ tệp Markdown nào thành PDF
+MarkdownToPDF.tags=đánh dấu,nội dung web,chuyển đổi,chuyển
+
+home.getPdfInfo.title=Lấy TẤT CẢ thông tin về PDF
+home.getPdfInfo.desc=Lấy bất kỳ và tất cả thông tin có thể về PDF
+getPdfInfo.tags=thông tin,dữ liệu,số liệu thống kê,thống kê
+
+home.extractPage.title=Trích xuất (các) trang
+home.extractPage.desc=Trích xuất các trang được chọn từ PDF
+extractPage.tags=trích xuất
+
+home.PdfToSinglePage.title=PDF sang một trang lớn
+home.PdfToSinglePage.desc=Ghép tất cả các trang PDF thành một trang lớn duy nhất
+PdfToSinglePage.tags=trang đơn
+
+home.showJS.title=Hiển thị Javascript
+home.showJS.desc=Tìm kiếm và hiển thị bất kỳ JS nào được chèn vào PDF
+showJS.tags=JS
+
+home.autoRedact.title=Tự động biên tập
+home.autoRedact.desc=Tự động biên tập (Che đen) văn bản trong PDF dựa trên văn bản đầu vào
+autoRedact.tags=Biên tập,Ẩn,che đen,đen,bút đánh dấu,ẩn
+
+home.tableExtraxt.title=PDF sang CSV
+Here's the translated Vietnamese version of the properties file:
```

Comment:
^
Here's the translated Vietnamese version of the properties file:
again bad copy paste here

Reply:
my bad, really sorry. Removed